pynws/simple_nws.py

```diff
@@ -138,6 +138,8 @@ async def set_station(self: SimpleNWS, station: Optional[str] = None) -> None:
             if not self.stations:
                 self.stations = [self.station]
         else:
+            if not (self.wfo and self.x and self.y):
+                await self.get_points()
```

Comment:
This is already handled inside `get_points_stations`.  Is there a use case to keep it here?